docs/console/docs/010-details-uiextensibility.md

```diff
@@ -0,0 +1,69 @@
+---
+title:  UI extensibility
+type: Details
+---
+
+The Kyma Console UI uses the [Luigi framework](https://github.com/kyma-project/luigi) to allow you to seamlessly extend the UI content with custom micro front-ends.
+
+## Console's Extensibility in a nutshell
```

Comment:
I don't know if the title really reflects what is in the paragraph. Although it includes extensibility because of the part about micro front-ends and their interaction with the Console. But I would say it's more about how the console UI works (how the navigation is rendered and how it works) and how it interacts with the micro front-ends to display them - but not how to extend the console ui with the micro front-end. (I hope this comment is understandable ;)

Maybe the title could be: `Console UI interaction with micro front-ends` ? I'm open to suggestions :)